README.md

```diff
@@ -20,7 +20,7 @@ import com.jakewharton.nopen.annotation.Open;
 
 final class Foo {}
 abstract class Bar {}
-@Open class Baz {}
+@Open class Bad {}
```

Comment:
This is meant to be `Baz` since it's a correct case. Open classes aren't intrinsically bad, they just need to be something you want.